api_app/analyzers_manager/migrations/0052_analyzer_config_dns0_names.py

```diff
@@ -0,0 +1,220 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "name": "DNS0_names",
+    "python_module": {
+        "module": "dns0.dns0_names.DNS0Names",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "description": "Run advanced searches on billions of current and historical domain names.",
```

Comment:
sorry another thing, can you add here  a link to the DNS0 service too? That would appear in the GUI as a consequence.

Reply:
Of course. you mean in the "description" field  ?